docs/specification.yaml

```diff
@@ -767,6 +802,32 @@ components:
           example: "This is a _new_ page's content in **markdown**!"
       required:
         - route
+    FindPagesRequestBody:
+      type: object
+      properties:
+        taxonomyFilter:
```

Comment:
This specification is not quite accurate. It currently implies a request body structure such as:

```json
{
  "taxonomyFilter": [
    {
      "custom_taxonomy_key": "author", 
      "custom_taxonomy_values": [
        "ksmith" 
      ] 
    } 
  ], 
  "operation": "or"
}

Reply:
Fixed it